packages/grafana-ui/src/components/uPlot/config.ts

```diff
@@ -175,6 +175,23 @@ export interface HideableFieldConfig {
   hideFrom?: HideSeriesConfig;
 }
 
+/**
+ * @alpha
+ */
+export enum StackingMode {
+  None = 'none',
+  Standard = 'standard',
+  Percent = 'percent',
+}
+
+/**
+ * @alpha
+ */
+export interface StackingConfig {
+  stackingMode?: StackingMode;
+  stackingGroup?: string;
+}
```

Comment:
I would try to not have have two stacking options, instead combine these into a single stacking (string) prop: 

stacking?: string;

valid values would be undefined (disabled), 'normal'  = normal stacking, we can add 'percent' as a special value later.  Or if any other value it's a custom stacking group.  

A bit similar to https://api.highcharts.com/highcharts/plotOptions.series.stacking

Reply:
Or a separate option is not so bad, hm.. tricky